src/test/java/jp/ikedam/jenkins/plugins/extensible_choice_parameter/FilenameChoiceListProviderSimpleTest.java

```diff
@@ -26,7 +26,8 @@
 import static org.junit.Assert.*;
 
 import java.io.File;
-import java.io.IOException;
+import java.io.IOException;
```

Comment:
This diff looks changing line break characters.
I worry that accepting this request results inconsistent line breaks.